tests/testthat/test-dupree_package_integration.R

```diff
@@ -36,7 +39,7 @@ test_that("`dupree_package` fails when passed a non-R package structure", {
   file.create(file.path(d, "DESCRIPTION"))
 
   expect_error(
-    dupree_package(d),
+    as.data.frame(dupree_package(d)),
```

Comment:
Error could occur in conversion to data.frame; the test should be `expect_error(dupree_package(args))`